bqplot/marks.py

```diff
@@ -1519,12 +1519,8 @@ class GridHeatMap(Mark):
     font_style = Dict().tag(sync=True)
 
     def __init__(self, **kwargs):
-        data = kwargs['color']
-        kwargs.setdefault('row', range(data.shape[0]))
-        kwargs.setdefault('column', range(data.shape[1]))
```

Comment:
should this have been `range(data.shape[0] + 1)` and `range(data.shape[1] + 1)` respectively?